packages/client/package.json

```diff
@@ -108,6 +108,7 @@
     "xregexp": "^4.2.0"
   },
   "devDependencies": {
+    "@opencrvs/gateway": "^1.5.0",
```

Comment:
does the client need this?

Reply:
It does now as I added the logic to look at workflow dependencies to figure out which packages cannot be optimised away in build and test processes. Client did seem to have a dependency to gateway for whatever reason

Reply:
I tried to find the dependency but couldn't :O